dev-lang/rust/files/1.32.0-libressl.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/vendor/openssl-sys/build/main.rs b/vendor/openssl-sys/build/main.rs
+index 2b1691230..69def400e 100644
+--- a/vendor/openssl-sys/build/main.rs
++++ b/vendor/openssl-sys/build/main.rs
+@@ -1,9 +1,9 @@
+ extern crate cc;
++#[cfg(feature = "vendored")]
++extern crate openssl_src;
+ extern crate pkg_config;
+ #[cfg(target_env = "msvc")]
+ extern crate vcpkg;
+-#[cfg(feature = "vendored")]
+-extern crate openssl_src;
```

Comment:
Why does this ordering matter?

Reply:
That's from https://github.com/sfackler/rust-openssl/commit/9fd7584a84168655cb27e03b7e19a9847b88e77f
The patch is called "Support LibreSSL 2.8.x", I don't know if these lines order matters.